README.md

```diff
@@ -23,7 +23,7 @@ A full example leveraging other community modules is contained in the [examples/
 ```hcl
 module "eks" {
   source                = "terraform-aws-modules/eks/aws"
-  version               = "0.1.0"
+  version               = "1.0.0"
```

Comment:
Good catch! Hmm, does sourcing from the registry work without a version at all? kind of a burden to remember this on every release.

Reply:
Yes it does, it will just grab the latest version if this is not included, I can remove this line altogether?

Reply:
Yes, that'd be better. Thanks!

Reply:
since it will not possible to have this feature in version 1.0.0, we should update it to 1.2.0 now or others later.

Otherwise, the reader will be confused.